### ISSUE TITLE
Account form hints need translation

### ISSUE THREAD

**Issue by vincius0000** [enrique-lozano/Monekin#226]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/enrique-lozano/Monekin/blob/main/CONTRIBUTING.md#reporting-issues).
- [X] I agree to follow the [Code of Conduct](https://github.com/enrique-lozano/Monekin/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/enrique-lozano/Monekin/issues?q=is%3Aissue) that already report this problem, without success.
- [X] I have updated the app to the last released version.

### Current Behavior

Within _account_form.dart_ some hint texts are hardcoded, ex:
`hintText: 'Ex.: My account'`

### Expected Behavior

Both 'Ex.:' and 'My account' words should be written in selected language.

### Steps to Reproduce

Start creating new account.

**Comment by vincius0000** [comment]
There are more I've found, import_csv.dart
```dart
return Column(
    children: [
      buildColumnSelector(
        value: notesColumn,
        labelText: 'Note column',
        headersToSelect: headersToSelect,
        onChanged: (value) {
          setState(() {
            notesColumn = value;
          });
        },
      ),
      const SizedBox(height: 16),
      buildColumnSelector(
        value: titleColumn,
        labelText: 'Title column',
        headersToSelect: headersToSelect,
        onChanged: (value) {
          setState(() {
            titleColumn = value;
          });
        },
      ),
    ],
  );
```

I've had the "Welcome back!" at home page mistranslated as well, I did have snackbar notification in wrong translation too after full export, but I'm looking into that on why that happen. The snackbar was in Spanish and I'm trying to see where it's configured like so.

**Comment by enrique-lozano** [comment]
Yep, some translations are missing. We need to add the keys to the translations JSON files and then run `dart dun slang`. After that, we can use the new translationa
